frontend/src/pages/modelServing/customServingRuntimes/DeleteCustomServingRuntimeModal.tsx

```diff
@@ -35,7 +35,8 @@ const DeleteCustomServingRuntimeModal: React.FC<DeleteCustomServingRuntimeModalP
       onDelete={() => {
         if (template) {
           setIsDeleting(true);
-          deleteTemplate(template.metadata.name, template.metadata.namespace)
+          // TODO: Revert back to pass through api once we migrate admin panel
+          deleteTemplateBackend(template.metadata.name, template.metadata.namespace)
```

Comment:
so by passing it though the api, you use the serviceaccount privileges?  Are you planning to extend the Openshift default ClusterRoles for project rolebindings, e.g Admin, View, Edit with custom Clusterroles, e.g. an Admin being able to delete Custom Serving Runtimes or to list storageclasses? Or do you go with the standard logged-in user roles  Admin, View, Edit ?